protocol/optyfi-protocol/vaults.md

```diff
@@ -1,12 +1,12 @@
 # Vaults
 
-Vaults are OptyFi smart contracts into which users deposit assets so that these assets can be deployed into strategies. Vaults enable users to share gas costs and thereby increase user profitability while also enabling more frequent strategy switching. 
+Vaults are OptyFi smart contracts into which users deposit assets in order to get these assets deployed into strategies. Vaults enable users to share gas costs and thereby increase user profitability while also enabling more frequent strategy switching. 
 
-Each vault has a [vault type](vaults.md#vault-types) determined by its underlying token \(i.e. the token users would deposit into the vault\). Each vault is also associated with a specific [risk profile](risk-framework.md#risk-profiles) which determines which strategies the vault can and cannot deploy assets into. 
+Each vault has a [vault type](vaults.md#vault-types) determined by its [underlying token](tokens-and-pools.md#tokens) \(i.e. the token users would deposit into the vault\). Each vault is also associated with a specific [risk profile](risk-framework.md#risk-profiles) which determines which strategies the vault can and cannot deploy assets into. 
```

Comment:
Recommendation:
...which determines which strategies the vault is permitted to deploy assets into.